CRM/MembershipExtras/Job/OfflineAutoRenewal/PaymentPlan.php

```diff
@@ -160,6 +162,27 @@ private function setContributionPendingStatusValue() {
     ]);
   }
 
+  /**
+   * Gets contribution Statuses Name to value Mapping
+   *
+   * @return array $contributionStatusesNameMap
+   */
+  private function setContributionStatusesNameMap() {
```

Comment:
I left a note about this before, I think you can just use `CRM_MembershipExtras_Service_ContributionUtilities` class here